ush/forecast_predet.sh

```diff
@@ -0,0 +1,299 @@
+#! /bin/sh
+
+##### 
+## "forecast_def.sh"
+## This script sets value of all variables
+##
+## This is the child script of ex-global forecast,
+## This script is a definition of functions.
+#####
+
+
+# For all non-evironment variables
+# Cycling and forecast hour specific parameters
+common_predet(){
+  echo "SUB ${FUNCNAME[0]}: Defining variables for shared through models"
+  pwd=$(pwd)
+  machine=${machine:-"WCOSS_C"}
+  machine=$(echo $machine | tr '[a-z]' '[A-Z]')
+  CASE=${CASE:-C768}
+  CDATE=${CDATE:-2017032500}
+  DATA=${DATA:-$pwd/fv3tmp$$}    # temporary running directory
+  ROTDIR=${ROTDIR:-$pwd}         # rotating archive directory
+  ICSDIR=${ICSDIR:-$pwd}         # cold start initial conditions
+}
+
+DATM_predet(){
+  SYEAR=$(echo  $CDATE | cut -c1-4)
+  SMONTH=$(echo $CDATE | cut -c5-6)
+  SDAY=$(echo   $CDATE | cut -c7-8)
+  SHOUR=$(echo  $CDATE | cut -c9-10)
+  # directory set up
+  if [ ! -d $DATA ]; then mkdir -p $DATA; fi
+  if [ ! -d $DATA/DATM_INPUT ]; then mkdir -p $DATA/DATM_INPUT; fi
+  FHMAX=${FHMAX:-9}
+  # Go to Run Directory (DATA)         
+  cd $DATA
+}
+
+FV3_GFS_predet(){
+  echo "SUB ${FUNCNAME[0]}: Defining variables for FV3GFS"
+  CDUMP=${CDUMP:-gdas}
+  CDUMPwave="${CDUMP}wave"
+  FHMIN=${FHMIN:-0}
+  FHMAX=${FHMAX:-9}
+  FHOUT=${FHOUT:-3}
+  FHZER=${FHZER:-6}
+  FHCYC=${FHCYC:-24}
+  FHMAX_HF=${FHMAX_HF:-0}
+  FHOUT_HF=${FHOUT_HF:-1}
+  NSOUT=${NSOUT:-"-1"}
+  FDIAG=$FHOUT
+  if [ $FHMAX_HF -gt 0 -a $FHOUT_HF -gt 0 ]; then FDIAG=$FHOUT_HF; fi
+  WRITE_DOPOST=${WRITE_DOPOST:-".false."}
+  restart_interval=${restart_interval:-0}
+  rst_invt1=`echo $restart_interval |cut -d " " -f 1`
+  
+  PDY=$(echo $CDATE | cut -c1-8)
+  cyc=$(echo $CDATE | cut -c9-10)
+  
+  # Directories.
+  pwd=$(pwd)
+  NWPROD=${NWPROD:-${NWROOT:-$pwd}}
+  HOMEgfs=${HOMEgfs:-$NWPROD}
+  FIX_DIR=${FIX_DIR:-$HOMEgfs/fix}
+  FIX_AM=${FIX_AM:-$FIX_DIR/fix_am}
+  export FIX_AER=${FIX_AER:-$FIX_DIR/fix_aer}
```

Comment:
export FIX_AER=${FIX_AER:-$FIX_DIR/fix_aer}
  export FIX_LUT=${FIX_LUT:-$FIX_DIR/fix_lut}
are new from f/c-c they're also exported while none of the other variables are?  Is there a reason for this?

Reply:
These are from the develop branch. @KateFriedman-NOAA @WalterKolczynski-NOAA Could you answer this question?

Reply:
We just discussed this a couple days ago. These do not need to be exported.

Reply:
Resolved.